example/src/containers/Dashboard/Dashboard.tsx

```diff
@@ -31,9 +37,20 @@ export const Dashboard = () => {
                 setAddress(await getAddress());
                 setBalance(await getBalance());
                 setLatestBlock(await getLatestBlock());
+
+                let api = await getPolkadotApi();
+                if (api) {
+                    api.on(PolkadotEvent.OnBalanceChange, balanceCallback)
+                }
             }
         })();
-    }, [state.polkadotSnap.isInstalled]);
+        return async function cleanup() {
+            if (state.polkadotSnap.isInstalled) {
```

Comment:
@mpetrunic can you help me, I don't understand why this `snap.polkadotSnap.isInstalled` is always false?